### PR TITLE
enable download dialog on main page

### PR DESCRIPTION
**Pull request by tewshi**
- [x] Adds cypress test scaffold
- [x] adds test for app download buttons
- [x] restores app download modal

### CODE REVIEW

File: tests/e2e/fixtures/example.json
Comment (kelsos) [inline tests/e2e/fixtures/example.json:1]: We don't really need this placeholder fixture right?

Comment (tewshi) [reply tests/e2e/fixtures/example.json:1]: yeah, not sure we will need to mock api calls for now... was part of the cypress scaffold, i'll remove it.

Comment (kelsos) [reply tests/e2e/fixtures/example.json:1]: We can do it later :)

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**